mapcss/MapCSSListenerL.py

```diff
@@ -184,8 +195,10 @@ def exitBooleanExpression(self, ctx:MapCSSParser.BooleanExpressionContext):
             'operands':
                 v['booleanExpressions'] or # Juste get operands array
                 (v['functionExpression'] and [v['functionExpression']]) or
-                []
+                [],
+            'selector_index': self.selector_index
```

Comment:
```suggestion
                'selector_index': self.selector_index
```

Reply:
Are you sure about this? It should have the same indentation as the other keys ('type', 'operator', 'operands') of the dictionary right?

Reply:
Right!